zubhub_backend/zubhub/creators/views.py

```diff
@@ -186,7 +188,6 @@ class RegisterCreatorAPIView(RegisterView):
             "location": "string",\n
             "bio": "",\n
             "subscribe": false\n
-            "creator_tags": "string",\n
```

Comment:
here you are specifying that `creator_tags` is a string, but you were treating it like an array everywhere else